src/styles/_catalog.scss

```diff
@@ -1,9 +1,33 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
   margin: 0 auto;
+  display: grid;
+  justify-content: center;
+
+  gap: 48px;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
More correctly to move repeated value to variable. To change the width of the card, it will be enough to change the value of the variable, and not search through the entire code